src/encoded/schemas/biosource.json

```diff
@@ -142,7 +151,7 @@
         "tissue.preferred_name": {
             "title": "Tissue"
         },
-        "individual.organism.name": {
+        "organism_name": {
```

Comment:
> Overall the changes look good, but we should take a close look at the embeds so that we're not adding unnecessary/duplicate embeds.
> 
> The main example here are places where `...biosource.organism.name` is embedded as well as `...biosource.individual.organism.name`. If there is an Individual on the Biosource, then `biosource.individual.organism` is the same item as `biosource.organism` and I think embedding the `name` property twice is unnecessary. For the sake of invalidation scope, the concern is that if one item changes (e.g. the Organism item), we want to make sure all items that embed a relevant property are invalidated so they get updated, but embedding the same db item twice doesn't help invalidation as far as I can tell.

@sbreiff I agree that this does need careful consideration as to not have unnecessary redundant embeds.  The reason that I left most if not all of the ones in question there was that almost always the calculated property uses the link from biosource to individual to organism to make the linkTo and therefore a change to the Individual item (by changing it's organism) should trigger invalidation of the containing items.  It was my understanding that to ensure appropriate invalidation of items based on calculated properties you needed to embed all the fields used in the calcprop - but it's been awhile so that may not be the case?

Reply:
the path to the item doesn't matter, just the itemtype and property. so having biosource.individual.organism.name is equivalent to biosource.organism.name. Biosource is the item that has the calculated property, so biosource will need to embed both, but experiment and experiment_set and biosample do not.

Reply:
OK - got it.  I think I've trimmed unnecessary/redundant embeds.